recipes/flann/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set revision = "1.8.4" %}
+
+{% set tag = environ.get("GIT_DESCRIBE_TAG", "") %}
+{% set number = environ.get("GIT_DESCRIBE_NUMBER", "0") %}
+{% set hash = environ.get("GIT_DESCRIBE_HASH", "0") %}
```

Comment:
Linter seems to complain that `environ` is not set. Could it at least have an empty dictionary? That would reproduce reasonable behavior and there is at least some effort to handle it here. Ultimately, the repo is needed to get the correct values here.